File: sdks/ruby/lib/cyberarmor/providers/amazon.rb
# frozen_string_literal: true

module CyberArmor
  module Providers
    class Amazon < OpenAICompatible
      def initialize(client:, openai:, tenant_id:, model: 'amazon.nova-lite-v1:0')
        super(client: client, openai: openai, tenant_id: tenant_id, provider: 'amazon', model: model)
      end
    end
  end
end

File: sdks/php/src/Providers/Amazon.php
<?php

declare(strict_types=1);

namespace CyberArmor\Providers;

use CyberArmor\Client;

final class Amazon extends OpenAICompatible
{
    public function __construct(Client $client, object $openai, string $tenantId, string $model = 'amazon.nova-lite-v1:0')
    {
        parent::__construct($client, $openai, $tenantId, 'amazon', $model);
    }
}

